bin/cypress/vars.sh

```diff
@@ -0,0 +1,7 @@
+export SECRET_WORDS="chief loud snack trend chief net field husband vote message decide replace"
+export CYPRESS_SECRET_WORDS=$SECRET_WORDS
+export NETWORK_NAME=localhost
+export PASSWORD="g1tc01nc0"
+export CYPRESS_REMOTE_DEBUGGING_PORT=9222
+export METAMASK_VERSION="9.8.1"
+export VERBOSE=1
```

Comment:
If we are no longer using MetaMask, these can all go away. They were required for the `cypress-metamask` lib